### ISSUE TITLE
Time conversion issue while converting JSON to Parquet.

### ISSUE THREAD

**Issue by Hasan-1999** [Cinchoo/ChoETL#144]
I am trying to convert JSON file to Parquet in Azure function using the below method:

```
string jsonFilePath = @"F:\contactdata.json";
string json = File.ReadAllText(jsonFilePath);
using (var reader = ChoJSONReader.LoadText(json))
{
     using (var writer = new ChoParquetWriter(F:\jsontoparquet.parquet))
       {
           writer.Write(reader);
        }
}
```

 But I'm getting exception while executing the above code, the error is:

>  System.Private.CoreLib: The UTC Offset of the local dateTime parameter does not match the offset argument. (Parameter 'offset').

**Comment by Cinchoo** [comment]
Pls share sample json file.

**Comment by Hasan-1999** [comment]
> Pls share sample json file.
[contactdata.txt](https://github.com/Cinchoo/ChoETL/files/6584051/contactdata.txt)

Please find the sample JSON file. 
Note: GitHub is not accepting json format file, so I have shared it as .txt format

**Comment by Cinchoo** [comment]
It is a bug in the library handling datatimeoffset. Applied fix, pushed new package v1.0.1.8. Try it. 

One thing I noticed from your sample code, missing json path to select the nodes. (otherwise the library will flatten out all the fields from root). I believe you want to convert sub nodes to parquet file.


Here is the sample

```
            using (var r = new ChoJSONReader("sample52.json")
                .WithJSONPath("$..data")
                )
            {
                using (var w = new ChoParquetWriter("myData52.parquet"))
                    w.Write(r);
            }
```

**Comment by Hasan-1999** [comment]
Thank You for your response. I tried exactly what you suggested, but this time I got this error:

> Exception while executing function: jsontoparquet. System.Private.CoreLib: The UTC time represented when the offset is applied must be between year 0 and 10,000. (Parameter 'offset').

Here is my code:
```
            using (var reader = ChoJSONReader.LoadText(@"F:\contactdata.json").WithJSONPath("$..data"))
            {
                using (var writer = new ChoParquetWriter(@"F:\jsontoparquet.parquet"))
                {
                    writer.Write(reader);
                }
            }
```
I'm looking forward to a solution to this issue.

**Comment by Cinchoo** [comment]
I couldn't reproduce it locally, But did quick search on internet, found a fix. Applied fix, pushed new package v1.0.1.9. Try and let me know. Hope the fix solve the issue.

**Comment by Hasan-1999** [comment]
> I couldn't reproduce it locally, But did quick search on internet, found a fix. Applied fix, pushed new package v1.0.1.9. Try and let me know. Hope the fix solve the issue.

Thank You for your response. Time Conversion issue is fixed now. I can convert JSON file to Parquet file now.
